urdf/rooster_standalone.urdf.xacro

```diff
@@ -18,8 +18,10 @@
   <xacro:include filename="$(find rooster_description)/urdf/rooster.urdf.xacro" />
   <xacro:arg name="simulation" default="false" />
   <xacro:arg name="ground_camera" default="false" />
+  <!-- Need to put the arithmetic computation into quotes because otherwise xacro fails to build -->
+  <xacro:arg name="ouster_rotation" default="'${-pi/4}'" />
```

Comment:
If you add the colon above, you can remove this argument here

Reply:
I doubt it will get used, but I'll keep this in case different rotation is necessary for the standalone